src/ray/core_worker/task_event_buffer.h

```diff
@@ -325,6 +327,14 @@ class TaskEventBufferImpl : public TaskEventBuffer {
   /// process them quick enough.
   std::atomic<bool> grpc_in_progress_ = false;
 
+  /// Profile events dropped per job, to be reported to GCS. Reported data loss
+  /// will be removed from this.
+  absl::flat_hash_map<JobID, uint32_t> profile_events_dropped_ GUARDED_BY(mutex_);
```

Comment:
We don't need to categorize this per job id. There can be only 1 job per worker.

Reply:
ah - right. Good catch.